### ISSUE TITLE
Review 'Accessibility Review' stories

### ISSUE THREAD

**Issue by richpomfret** [FreeUKGen/FreeUKRegProductIssues#1117]
Establish what's been completed as part of other stories.

STAS-F = Fail' at AA or A
STAS-A = needed to Achieve AAA

Priority scoring (as an epic for completion)
Cost 6 ( effort of 5 to review what has been done)
Economic 0
Strategic importance  10 (past milestone)
Urgency 3 (on the basis that it feels like more than a 4)
Total score: 19
